README.md

```diff
@@ -37,6 +38,26 @@ Amber is packaged in the following distros:
 
 Arch (AUR) - `amber-bash-bin`
 
+#### Nix
+
+The package is already containing all the required install scripts and dependencies. You can use the flake as:
+
+```nix
```

Comment:
is all that info really necessary? if thats just a package, you should mention it with one line under the "Via package manager" header:

Nix - `amber-lang`

if you really need this info, you could wrap it in a dropdown box so it doesnt take up much space

<details>
<summary>like this</summary>

```
<details>
<summary>Title</summary>

### Put stuff here
</details>
```
</details>

im just saying that not every amber user is a nix user. they don't need this information.

Reply:
also nothing is preventing you from creating a `NIX.md` file near `README.md` with the necessary instructions